static/bidder-info/nativo.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "https://exchange.postrelease.com/esi?ntv_epid=7"
```

Comment:
"esi?ntv_epid=7" does not look fine

Reply:
this is correct, that's the url to our exchange server, in this case, prebid server.